front/services/address/AddressForm.js

```diff
@@ -123,7 +123,7 @@ export default class AddressForm {
         addClass(select, 'form__field');
         addClass(select, 'em-form__field');
 
-        insertAfter(this._city, select);
+        insertAfter(this._cityName, select);
```

Comment:
why do we change this?

Reply:
If need to change this because the field is not rendered at the good place. Can't found an other way to do it